doc/chapters/gateways/10-environment-gateway.rst

```diff
@@ -74,4 +74,14 @@ to e.g. ``/tmp/test``, the varaiable value would now be set to ``/tmp/test:/some
 If ``SOME_ENVIRONMENT_VARIABLE`` had not been previously set, the value would now be
 set to ``:/some/path``.
 
+Important Note
+--------------
 
+Environment variables are internally stored as an array of pointers to strings in Linux based
+systems. This array is terminated by a NULL pointer. Each name and value pairs are matched with "="
```

Comment:
it is possible but i think we do not need to do it. Cause the sentence should give concise reasoning that why we hould not use those special characters. If the sentence is not enought then i should develop it i guess.